### PR TITLE
Group member list screen UI

### LINKED ISSUE #44: group members list screen UI
**Is your feature request related to a problem? Please describe.**
As an admin of the org I would like to view the members of the groups under the org. 

image

**Describe the solution you'd like**
1. create group members list screen UI
1. should be alphabetically ordered similar to a contacts app.
1. Divided into alphabetic sections

### PR DESCRIPTION
**Pull request by divy-arpit**
### Description

This PR adds to the app a screen which would show the members of a group

**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Tested on local device.

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance**
- [ ] My code follows TDD clean architecture
- [X] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Screenshot_1591164310

### CODE REVIEW

File: lib/screens/group_member_list/widgets/create_list_tile.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+///This class returns a list tile which would hold the name and details of member
+///It receives a string as parameter which is the name of the person
+
+class ListTileCreate extends StatelessWidget {
```
Comment (naveennvrgup) [inline lib/screens/group_member_list/widgets/create_list_tile.dart]:
```suggestion
class GroupMemberTile extends StatelessWidget {
```

File: lib/screens/group_member_list/widgets/create_list_tile.dart [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  final String title;
+
+  ListTileCreate(this.title);
```
Comment (naveennvrgup) [inline lib/screens/group_member_list/widgets/create_list_tile.dart]: 1. Please take in profilePicUrl(optional) and memberId(compulsory) also. 
2. If the profilePicUrl is null then show account_circle else pic from that URL
3. Use named arguments :innocent:

**Review by naveennvrgup — CHANGES_REQUESTED** [review]
1. Great job with PR @divy-arpit  :fire:
1. nice to see PR from a separate branch

File: lib/screens/group_member_list/widgets/create_list_tile.dart (hunk 1 above)
Comment (naveennvrgup) [reply lib/screens/group_member_list/widgets/create_list_tile.dart]: also please rename the file accordingly

File: pubspec.yaml [hunk 3]
```diff
@@ -28,6 +28,7 @@ dependencies:
   google_fonts: ^0.5.0+1
   # Picking images from the image library
   image_picker: ^0.6.6+5
+  transparent_image: ^1.0.0
```
Comment (naveennvrgup) [inline pubspec.yaml]: I think it provides redudant functionality. I suggest you try the flutter's built in fadeinimg widget.  
https://api.flutter.dev/flutter/widgets/FadeInImage-class.html
```
      return FadeInImage(
        placeholder: AssetImage("assets/images/icon_ecell.png"),
        image: NetworkImage(this.profilePicUrl),
      );
```

File: lib/screens/group_member_list/widgets/group_member_tile.dart [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import 'package:flutter/material.dart';
+import 'package:transparent_image/transparent_image.dart';
+
+///This class returns a list tile which would hold the name and details of member
+///It receives a 'title' as a required named argument which is the name of the member and url of profile image as a named argument
+
+class GroupMemberTile extends StatelessWidget {
+  final String title, profilePicUrl;
+
+  GroupMemberTile({@required this.title, this.profilePicUrl});
```
Comment (naveennvrgup) [inline lib/screens/group_member_list/widgets/group_member_tile.dart]: sorry to mention it late you can take in member id also :sweat_smile:

**Review by naveennvrgup — CHANGES_REQUESTED** [review]

File: lib/screens/group_member_list/widgets/group_member_tile.dart (hunk 4 above)
Comment (divy-arpit) [reply lib/screens/group_member_list/widgets/group_member_tile.dart]: I assume that member ID is going to be an integer and so I have added functionalities accordingly sir.

**Review by naveennvrgup — APPROVED** [review]

**Pull request merged**